programs/CMakeLists.txt

```diff
@@ -47,8 +47,8 @@ if(LIBDEFLATE_BUILD_GZIP)
         set(GZIP ${CMAKE_INSTALL_FULL_BINDIR}/libdeflate-gzip)
         set(GUNZIP ${CMAKE_INSTALL_FULL_BINDIR}/libdeflate-gunzip)
```

Comment:
Also, please put the `${CMAKE_EXECUTABLE_SUFFIX}` here in these `set` commands, instead of embedded in the `install` command.

Reply:
Yes, that makes more sense